### DISCUSSION TITLE
gouravkhunger/adding-multiple-authors-to-a-jekyll-blog-got-easier

### DISCUSSION THREAD

**Discussion by giscus** [genicsblog/comments#131] [Announcements]
# gouravkhunger/adding-multiple-authors-to-a-jekyll-blog-got-easier

Jekyll doesn't support multiple authors out of the box. But we can add that functionality using this plugin I made!

https://genicsblog.com/gouravkhunger/adding-multiple-authors-to-a-jekyll-blog-got-easier

**Comment by ciroiriarte** [comment]
Hello!, can it be used on Github pages?.  Repo intro for jekyll-paginate-v2 is not supported there: https://github.com/sverrirs/jekyll-paginate-v2

**Reply by gouravkhunger** [reply]
Hi there @ciroiriarte, thank you for taking the time to comment!

This plugin, like all other custom jekyll plugins, does not work with GitHub Pages out of the box. Since only certain versions of [whitelisted plugins](https://pages.github.com/versions) are available in the GitHub Pages environment, you will need to either build your site locally and deploy the output folder, or set up GitHub Actions to do that for you, for example each time you push a commit.

I'll recommend using the GitHub Actions approach. The [plugin repository](https://github.com/gouravkhunger/jekyll-auto-authors) has an [example blog](https://github.com/gouravkhunger/jekyll-auto-authors/tree/main/example) including [a GitHub Action](https://github.com/gouravkhunger/jekyll-auto-authors/blob/main/.github/workflows/publish-site.yml) that builds and deploys the site to GitHub Pages, you are free to take insights and come up with a similar build CI for yourself.

If you need any help, please don't hesitate to reach out through [our discord server](https://discord.genicsblog.com)!
